cmd/worker/main.go

```diff
@@ -78,6 +79,16 @@ func HandleKeyGeneration(ctx context.Context, t *asynq.Task) error {
 		"keyEDDSA": keyEDDSA,
 	}).Info("Key generation completed")
 
+	redis, err := storage.NewRedisStorage(config.AppConfig)
+	if err != nil {
+		return fmt.Errorf("storage.NewRedisStorage failed: %v: %w", err, asynq.SkipRetry)
+	}
+
+	err = redis.RemoveVaultCacheItem(ctx, fmt.Sprintf("vault-%s-%s", p.Name, p.SessionID))
+	if err != nil {
+		return fmt.Errorf("redis.RemoveVaultCacheItem failed: %v: %w", err, asynq.SkipRetry)
```

Comment:
Should we ignore keygen result if  RemoveVaultCacheItem failed?
Maybe its better to log it and ignore error return

Reply:
agree 👍